fastify.js

```diff
@@ -7,7 +7,8 @@ const https = require('https')
 const pinoHttp = require('pino-http')
 const Middie = require('middie')
 const fastseries = require('fastseries')
-const shot = require('shot')
+let shot
```

Comment:
Please use `var` for consistency.
Maybe we can set it as `null` to be more descriptive.